backend/services/battles.py

```diff
@@ -36,30 +34,31 @@ def battle_round(creator_pokemon, opponent_pokemon):
 
 def get_battle_winner(battle):
     battle_score = {"creator": 0, "opponent": 0}
-    battle_info = Battle.objects.filter(id=battle.id).values()[0]
 
-    # get the pokemons ids from the battles
-    creator_pokemons_id = [battle_info["creator_pokemon_" + str(i) + "_id"] for i in range(1, 4)]
-    opponent_pokemons_id = [battle_info["opponent_pokemon_" + str(i) + "_id"] for i in range(1, 4)]
-    # get the pokemons from the DB
-    creator_pokemon_list = [Pokemon.objects.filter(id=j).values()[0] for j in creator_pokemons_id]
-    opponent_pokemon_list = [Pokemon.objects.filter(id=j).values()[0] for j in opponent_pokemons_id]
+    # get the batween the battle and the trainers
+    creator = Team.objects.filter(battle=battle, trainer=battle.creator.id)
```

Comment:
Maybe this should be an `objects.get`? Same on the line right below.

Reply:
It errors if I change from filter to get. It shows me: 'Team' object is not subscriptable

Reply:
Because below you're doing `creator[0]`. So you need to change to `creator`